phpBB/includes/acp/acp_extensions.php

```diff
@@ -180,19 +180,13 @@ function main($id, $mode)
 					redirect($this->u_action);
 				}
 
-				if (confirm_box(true))
-				{
-					redirect($this->u_action . '&amp;action=enable&amp;ext_name=' . urlencode($ext_name) . '&amp;hash=' . generate_link_hash('enable.' . $ext_name));
-				}
-				else
-				{
-					confirm_box(false, $this->user->lang('EXTENSION_ENABLE_CONFIRM', $md_manager->get_metadata('display-name')), build_hidden_fields(array(
-						'i'			=> $id,
-						'mode'		=> $mode,
-						'action'	=> 'enable_pre',
-						'ext_name'	=> $ext_name,
-					)));
-				}
+				$this->tpl_name = 'acp_ext_enable';
+
+				$this->template->assign_vars(array(
+					'PRE'				=> true,
```

Comment:
It would preferrable to have a more descriptive name than just `PRE`, I suppose you're referring to the `pre_*` actions. And usually switches are prefixed with `S_`.